lib/schema/operations.graphql

```diff
@@ -0,0 +1,56 @@
+query GetAuthenticatedUser {
+    getAuthenticatedUser {
+        id
+        email
+        fullName
+        avatarUrl
+        userHandle
+    }
+}
+
+query GetUserProfileInfo {
```

Comment:
I'll remove this and incorporate `profileCompletionPercentage` into the `getAuthenticatedUser` query after we merge this, in PR #38

Reply:
Ah sure let me know if there are other conflicts I can resolve now